### DISCUSSION TITLE
Warning regarding missing field in the LIS PARAM Module with Noah-MP.4.0.1

### DISCUSSION THREAD

**Discussion by jayaisro** [NASA-LIS/LISF#1618] [Q&A] [answered]
Greetings!

I am a new user of the LIS and when trying to run the LIS module with Noah-MP.4.0.1, the file is showing warnings as shared in the log file. The parameters like ALBDRY1/2, ALBICE1/2, RSURF_SNOW etc are not found in the LIS param file, but may be important for my analysis. Would appreciate any help in this regard.

Thanks
[lislog_warnings.txt](https://github.com/user-attachments/files/17305318/lislog_warnings.txt)

**Comment by dmocko** [comment, marked as answer]
Hi @jayaisro 

Welcome to the LIS community.

Would you mind introducing yourself (full name and affiliation) as a professional courtesy?  And also please quickly let us know how you are planning to use the LIS software?

These warning messages with the Noah-MP-4.0.1 LSM are a little misleading.  These parameters would only be found in the LIS param file **IF** the user had used the LIS-OPT (optimization) sub-system to optimize these parameters.  Typically, without optimization, these parameters are just read from the Noah-MP-4.0.1 TBL files (MPTABLE.TBL, SOILPARM.TBL, GENPARM.TBL).

Unless you are using LIS-OPT, you can ignore these warnings.
